streaming/.eslintrc.cjs

```diff
@@ -1,4 +1,8 @@
+/* eslint-disable import/no-commonjs */
+
 // @ts-check
+
+// @ts-ignore - This needs to be a CJS file (eslint does not yet support ESM configd), and TS is complaining we use require
```

Comment:
Maybe just remove the ts-check?

Reply:
It is still useful so you get typecheck on `defineConfig`